svcb/svcb/build.py

```diff
@@ -252,12 +252,19 @@ def generateCMakeDecls(benchmarkObjs, sourceRootDir, supportedArchitecture, depe
             declStr += "  {}\n".format(macroName)
         declStr += ")\n"
       # Emit compiler flags
-      declStr += "{indent}target_compile_options({target_name} PRIVATE ${{SVCOMP_STD_{lang_ver}}})\n".format(
-        indent=cmakeIndent,
-        target_name=targetName,
-        lang_ver= lang_ver,
-        arch = arch.upper()
-      )
+      if coverage:
```

Comment:
Rather than duplicating code that handles `${{SVCOMP_STD_{lang_ver}}}` you should probably just add something like this as you are allowed in CMake to write `target_compile_options()` multiple times.

```python
if coverage:
  declStr += "{indent}target_compile_options({target_name} PRIVATE \"-fprofile-dir={profile_dir}.cov\")\n".format(
    indent= cmakeIndent,
    target_name = targetName,
    profile_dir = targetName
  )
```

Also as your're just using a relative path for the `profile_dir` where will the gcov files end up?